src/controller/CHIPDeviceControllerFactory.h

```diff
@@ -70,6 +70,7 @@ struct FactoryInitParams
 {
     FabricStorage * fabricStorage                                 = nullptr;
     System::Layer * systemLayer                                   = nullptr;
+    PersistentStorageDelegate * storageDelegate                   = nullptr;
```

Comment:
Perhaps `fabricIndependentStorage` or `allFabricsStorage` to make it clear what the semantics of this parameter are?

Reply:
fabricIndependentStorage seems good to me